MAUI/DataForm/floating-label-layout.md

```diff
@@ -0,0 +1,777 @@
+
+
+{% tabs %}
+{% highlight XAML %}
+<ContentPage 
+...
+xmlns:dataForm="clr-namespace:Syncfusion.Maui.DataForm;assembly=Syncfusion.Maui.DataForm">
+<dataForm:SfDataForm x:Name="dataForm" 
+                        LayoutType="TextInputLayout"
+                        DataObject="{Binding ContactInfo}">
+<dataForm:SfDataForm.Items>
+    <dataForm:DataFormPasswordItem FieldName="Name" 
+                                EnablePasswordVisibilityToggle="True">
+    </dataForm:DataFormPasswordItem>
+</dataForm:SfDataForm.Items>
+</dataForm:SfDataForm>
+</ContentPage>
+{% endhighlight %}
+{% highlight C# %}
+
+this.dataForm.GenerateDataFormItem += OnGenerateDataFormItem;
+
+private void OnGenerateDataFormItem(object sender, GenerateDataFormItemEventArgs e)
+{
+    if (e.DataFormItem != null && e.DataFormItem.FieldName == "Name" && e.DataFormItem is DataFormPasswordItem passwordItem)
+    {
+        passwordItem.LayoutType = DataFormLayoutType.TextInputLayout;
+        passwordItem.EnablePasswordVisibilityToggle = true;
+    }
+}
+
+{% endhighlight %}
+{% endtabs %}
+
+## Assistive label
+
+#### Hint label
+The data field name will be displayed as floating or hint label for the editor, and it will be enabled by using the `Name` or `LabelText` property.
+
+#### Helper label
+The helper label is used to display the water mark for the editor to provide hint for users and it can be set using `Prompt` or `PlaceHolderText`. 
+
+The visibility of helper text in DataForm floating label layout for `DataForm` and `DataFormItem` can be collapsed by setting `ShowHelperText` property of `TextInputLayoutSettings` to `false`
+
+{% tabs %}
+{% highlight XAML %}
+<ContentPage 
+...
+xmlns:dataForm="clr-namespace:Syncfusion.Maui.DataForm;assembly=Syncfusion.Maui.DataForm">
+<dataForm:SfDataForm x:Name="dataForm" 
+                        LayoutType="TextInputLayout"
+                        DataObject="{Binding ContactInfo}">
+    <dataForm:SfDataForm.TextInputLayoutSettings>
+        <dataForm:TextInputLayoutSettings ShowHelperText="False"/>
+    </dataForm:SfDataForm.TextInputLayoutSettings>
+</dataForm:SfDataForm>
+</ContentPage>
+{% endhighlight %}
+{% highlight C# %}
+
+this.dataForm.GenerateDataFormItem += OnGenerateDataFormItem;
+
+private void OnGenerateDataFormItem(object sender, GenerateDataFormItemEventArgs e)
+{
+    if (e.DataFormItem != null && e.DataFormItem.FieldName == "Name")
+    {
+        e.DataFormItem.LayoutType = DataFormLayoutType.TextInputLayout;
+        e.DataFormItem.TextInputLayoutSettings = new TextInputLayoutSettings
+        {
+            ShowHelperText = false,
+        };
+    }
+}
+
+{% endhighlight %}
+{% endtabs %}
+
+#### Validation label
+The validation label used to display the dataform validation messages such as valid or invalid data. Refer [validation](https://help.syncfusion.com/maui/dataform/validation) to learn more about dataform validation.
```

Comment:
The validation label displays the dataform validation messages such as valid or invalid data. Refer to the [validation](https://help.syncfusion.com/maui/dataform/validation) to learn more about the dataform validation.

Reply:
Content changed